google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITResultSetGetValue.java

```diff
@@ -615,9 +793,16 @@ public void testReadFloat64LiteralsGoogleStandardSQL() {
   public void testReadFloat64LiteralsPostgreSQL() {
     Assume.assumeTrue(dialect.dialect == Dialect.POSTGRESQL);
     try (ResultSet resultSet =
-        databaseClient.singleUse().executeQuery(Statement.of("SELECT 10.0 AS float64"))) {
+        databaseClient
```

Comment:
Why does this need a separate test case?

Reply:
same as above